templates/tests/server-test.yaml

```diff
@@ -35,6 +37,9 @@ spec:
           fi
 
           exit 0
-
+      volumeMounts:
+        {{ template "vault.mounts" . }}
```

Comment:
This drags in the `audit` and `data` volumes which doesn't make sense... shooting for something simpler